### DISCUSSION TITLE
Support Plans Console Pages UI Refresh (Office hours: Oct. 13, 2022)

### DISCUSSION THREAD

**Discussion by tonge495** [twilio-labs/paste#2734] [Office hours] [answered]
The specific questions I have are...
- What are potential design solutions for structuring a table so that certain columns are highlighted?
- How do I make the delineation between columns more apparent?
- The top header and left most column on the table both have icons, but one is for tooltips and the other opens a modal.  How do we visually distinguish between the two?  There are different situations where we use a tooltip vs a modal for additional description info on the table, but is there a more consistent way to handle this?
Background: 
Our team is doing a UI refresh for a few pages on Support Plans within Console.  This will be a 1:1 mapping from the existing legacy code to Paste components.
Stage of design: Exploration
Research: N/A
Links to mockups: View Support Plans Initial: https://www.figma.com/proto/ySsGmuuqxkAfWU30v7aTaH/Support-Plans-Migration?node-id=1%3A135918&scaling=min-zoom&page-id=0%3A1&starting-point-node-id=1%3A135918&show-proto-sidebar=1

**Comment by TheSisb** [comment, marked as answer]
Hey @tonge495,

Thanks for coming to office hours, glad we could be of help!

We're having some technical difficulties with the recording, but we'll share it in this thread when it becomes available.
